README.md

```diff
@@ -1,50 +1,56 @@
 [![Build Status](https://travis-ci.org/Xevib/osmbot.svg?branch=master)](https://travis-ci.org/Xevib/osmbot)
+
 OSMbot: OpenStreetMap bot for Telegram
 ======================================
-**OSMbot** is a project from the OpenStreetMap Catalan Community. See [WikiProject Catalan/OSMbot](http://wiki.openstreetmap.org/wiki/Ca:WikiProject_Catalan/OSMbot).
+**OSMbot** is a project maintained by the OpenStreetMap Catalan Community. See [WikiProject Catalan/OSMbot](http://wiki.openstreetmap.org/wiki/Ca:WikiProject_Catalan/OSMbot).
 
 Links
 -----
 
-* The bot: https://telegram.me/osmbot . You need Telegram in your smartphone or [Telegram Desktop](https://desktop.telegram.org/) in your PC (available for Linux, OS X and Windows). More about to get Telegram in [Getting Telegram](https://github.com/Xevib/osmbot/wiki/Getting-Telegram) page on the wiki.
+* The *Bot*: https://telegram.me/osmbot. In order to use the *Bot* you need to install a Telegram client in your smartphone or [Telegram Desktop](https://desktop.telegram.org/) in your PC (available for Linux, OS X and Windows). You can find more information about how to install Telegram in the [[Getting Telegram]](https://github.com/Xevib/osmbot/wiki/Getting-Telegram) page on the wiki.
 
```

Comment:
@Xevib I also suggest to move this hyperlink to the top of the project in order to have some like this:
`OpenStreetMap Telegram bot: https://telegram.me/osmbot`

Most of the projects promote their docs, or in this case the url to the bot in this way.